### DISCUSSION TITLE
有问题的话可以发布在此处，我会酌情回答

### DISCUSSION THREAD

**Discussion by shuo-liu16** [shuo-liu16/CS61A#3] [Announcements]
有问题的话可以发布在此处，我会酌情回答

**Comment by chenchen7427** [comment]
想问一下作者看的是哪个版本的视频呀，因为sp24的视频油管上好像没有（所以我目前的方案是fa20的视频配sp24的hw和lab）

**Reply by shuo-liu16** [reply]
这个课程比较简单，我觉得不是很需要视频，b站有视频的，有需要的话，你可以看看。
本教程有配套的课件，视频，电子式
https://www.learncs.site/docs/curriculum-resource/cs61a

**Reply by shuo-liu16** [reply]
我本人没怎么看视频，大多是看书，sp24找不到可能是因为这些课程仅向该校学生开放，也可能是过期了。

b站有视频，你可以看看

**Reply by lyu427** [reply]
Scheme项目的第13题写复杂了，直接
            if clause.rest == nil:
                return test
            while clause.rest != nil:
                return eval_all(clause.rest, env)
就可以
